src/common/reports/report_service.go

```diff
@@ -18,12 +18,17 @@ type ReportService struct {
 	report Report
 }
 
-const (
-	colorReset  = "\033[0m"
-	colorGreen  = "\033[32m"
-	colorYellow = "\033[33m"
-	colorBlue   = "\033[34m"
-	colorPurple = "\033[35m"
+var (
+	//colorReset  = "\033[0m"
+	//colorGreen  = "\033[32m"
+	//colorYellow = "\033[33m"
+	//colorBlue   = "\033[34m"
+	//colorPurple = "\033[35m"
+	colorReset  = ""
+	colorGreen  = ""
+	colorYellow = ""
+	colorBlue   = ""
+	colorPurple = ""
```

Comment:
@goody-cvs Can we turn this into a `struct`, and then when creating a new instance we will pass in the bool `noColor` and accordingly set all values? I don't like the global var...

Reply:
+1, And we won’t need the new func `noColorCheck`